polindrom.js

```diff
@@ -0,0 +1,23 @@
+//введеное слово полиндром или нет?
+let slovo = prompt('Введите слово без пробелов маленькими буквами','шалаш');
+let slovoArr = [];
+
+for (let i = 0; i < slovo.length; i++) { // заполняем массив буквами из введеного слова
+    slovoArr [i] = slovo [i];
+}
+
+let arrLength = slovoArr.length; //запомниаем количество букв в слове
+let y = 0; //счетчик совпадающих букв
+let n = Math.floor(slovoArr.length / 2);//разделим кол-во букв на 2 и округлим вниз (на случай нечетного кол-ва букв)
```

Comment:
that so complicated. Can you revert your word by for cycle. And compare it with original?